staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/conversion.go

```diff
@@ -369,3 +372,14 @@ func Convert_Slice_string_To_v1_IncludeObjectPolicy(in *[]string, out *IncludeOb
 	}
 	return nil
 }
+
+// Convert_Slice_string_To_v1_Preconditions allows converting a URL query parameter value.
+func Convert_Slice_string_To_v1_Preconditions(in *[]string, out *Preconditions, s conversion.Scope) error {
+	if len(*in) > 0 {
+		u := types.UID((*in)[0])
+		*out = Preconditions{UID: &u}
```

Comment:
Hmm - is that really what we want? Should we really ignore ResourceVersion?

@liggitt - for thoughts